### ISSUE TITLE
"Your Place In The World" (4641) has whitespace chars in Objectives text

### ISSUE THREAD

**Issue by Muehe** [cmangos/classic-db#234]
There are 4 chars at the end of the string that have no purpose, plus one space preceding them. Seem to be carriage return chars or something.

Fix:
```sql
UPDATE `quest_template` SET `Objectives`='Speak with Gornek. You recall Kaltunk marking your map with his location and mentioning that Gornek resided in the Den, a building to the west.' WHERE `entry`='4641';
```

**Comment by cala** [comment]
I remember that extra whitespaces in quest texts are a common thing and can be found in sniffs. Summoning @Tobschinski for his opinion on this matter.

**Comment by Tobschinski** [comment]
```Speak with Gornek. You recall Kaltunk marking your map with his location and mentioning that Gornek resided in the Den, a building to the west. ```

This is what I have. There is an extra whitespace character at the end.

**Comment by cala** [comment]
👍

**Comment by Muehe** [comment]
Well I have no problem with the space there, but it has some line-break chars too: `. \n\r\b\r`. AFAIK those should either not be there or be transformed to wildcards (`$B`).

**Comment by Muehe** [comment]
P.S.: Yeah it was changed in this commit: https://github.com/cmangos/classic-db/commit/548fbc55485dfa3beff5cfcdaf7a16604cd10027

**Comment by Muehe** [comment]
Closing here, as this would be fixed by the solution proposed in #237
